.github/workflows/ci.yml

```diff
@@ -348,6 +348,8 @@ jobs:
       #   # TODO do this less hackily
       - name: Build
         run: pnpm -r compile
+      - name: Build json schema
+        run: pnpm build-json-schema
```

Comment:
- let's instead have a step where we ensure that the checked in `isograph-config-schema.json` file is unchanged after we run `build-json-schema`, and have this run as a dependency of `all-checks-passed`